stable/gpu-feature-discovery/templates/rbac.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:
+  name: {{ include "gpu-feature-discovery.fullname" . }}
+  labels:
+{{ include "gpu-feature-discovery.labels" . | indent 4 }}
+rules:
+- apiGroups:
+    - ""
+  resources:
+    - services
+    - endpoints
```

Comment:
GPU feature discovery should lable nodes like NFD, no? Doesn't it need node get, patch, update as well?

Reply:
GPU feature discovery creates only additional list of labels for NFD, so doesn't need

Reply:
Cool